osf/models/user.py

```diff
@@ -7,6 +7,7 @@
 from copy import deepcopy
 from flask import Request as FlaskRequest
 from framework import analytics
+from os.path import splitext
```

Comment:
Minor: Follow pep8 conventions for import order/grouping: stdlib, then 3rd party, then internal.